readme.md

```diff
@@ -1 +1,80 @@
-# MAW11_Looper_GEJ
\ No newline at end of file
+# MAW11_Looper_GEJ 
+## Description
+
+This is a project for the MAW11 course. We're supposed to rewrite [this website](https://maw-looper.mycpnv.ch)
+
+## Getting Started
+### Prerequisites
+
+* Docker 4.33.0
+* PHP 8.3.11
+* Xdebug v3.3.2
+* postgres 16.4
+* pgadmin 8.11.0
+
```

Comment:
add composer version and git flow version

Reply:
why git flow ?